src/Compilers/CSharp/Portable/Binder/Binder_Constraints.cs

```diff
@@ -98,7 +99,7 @@ internal partial class Binder
 
             TypeParameterConstraintClause.AdjustConstraintTypes(containingSymbol, typeParameters, results, ref isValueTypeOverride);
 
-            RemoveInvalidConstraints(typeParameters, results, syntaxNodes, diagnostics);
+            RemoveInvalidConstraints(typeParameters, results!, syntaxNodes, diagnostics);
```

Comment:
>results [](start = 53, length = 7)

why this suppression?

Reply:
`RemoveInvalidConstraints` takes a non-null arraybuilder. We started with some nulls in the array, but just went through the whole thing to remove it.

---
In reply to: [367182789](https://github.com/dotnet/roslyn/pull/40946#discussion_r367182789) [](ancestors = 367182789)